<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabContract">
       <attribute name="title">
        <string>合同</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="layoutContract">
          <item row="1" column="0" colspan="3">
           <widget class="QTableView" name="tableViewContract"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="searchContractButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditContract"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="allContractButton">
            <property name="text">
             <string>查看所有</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="addContractButton">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editContractButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteContractButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHouse">
       <attribute name="title">
        <string>房屋</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="layoutHouse">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditHouse"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="allHouseButton">
            <property name="text">
             <string>查看所有</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QTableView" name="tableViewHouse"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="searchHouseButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="addHouseButton">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editHouseButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteHouseButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPerson">
       <attribute name="title">
        <string>个人</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="layoutPerson">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditPerson"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="allPersonButton">
            <property name="text">
             <string>查看所有</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="searchPersonButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QTableView" name="tableViewPerson"/>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="addPersonButton">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editPersonButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deletePersonButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
